src/Symfony/Component/Form/Extension/Validator/Type/FormTypeValidatorExtension.php

```diff
@@ -58,9 +60,17 @@ public function configureOptions(OptionsResolver $resolver)
             'constraints' => array(),
             'invalid_message' => 'This value is not valid.',
             'invalid_message_parameters' => array(),
+            'improved_validation_messages' => $this->improvedValidationMessages,
```

Comment:
Should we also add allowed types to the resolver for this option?

Reply:
Added